src/components/Breeds.js

```diff
@@ -29,7 +42,7 @@ const Breeds = () => {
   }
 
   function handleClick() {
-    const randomBreed = options[Math.floor(Math.random() * options.length)];
+    const randomBreed = breedsName[Math.floor(Math.random() * breedsName.length)];
```

Comment:
I recommend that using Lodash is quite useful in this case. Lodash.sample(options)